packages/ui-popover/src/Popover/index.tsx

```diff
@@ -466,6 +466,18 @@ class Popover extends Component<Props> {
         this._focusRegion.deactivate()
       }
     }
+
+    // if the props the state is based on change,
```

Comment:
I don't understand what you mean by this, can you please clarify this comment?

Reply:
Rephrased the comment, is it clearer now?

Reply:
looks good :)